template/.eslintrc.yml

```diff
@@ -41,6 +41,8 @@ rules:
     - error
     - namedComponents: arrow-function
       unnamedComponents: arrow-function
+  import/no-anonymous-default-export: 'off'
```

Comment:
I think we always prefer to have named exports. Is this needed explicitly for anything related to Storybook?